3.0-RELEASE-NOTES.md

```diff
@@ -49,3 +49,31 @@ without any callback, a promise is returned.
 
 See [pull request 918](https://github.com/strongloop/loopback-datasource-juggler/pull/918)
 for more details.
+
+
+##Remove deprecated model hooks and model events
+The following deprecated model hooks and events is now removed. Please use operations hooks in place
+of model hooks.
+
+###Model hooks
+beforeValidate
+afterValidate
+beforeCreate
+afterCreate
+beforeSave
+afterSave
+beforeUpdate
+afterUpdate
+beforeDestroy
+afterDestroy
```

Comment:
Could you please add a note that `afterInitialize` is staying around, because it's not a typical (async) model hook, but a just sync method.